ClientCore/Extensions/RampastringToolsExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+
+using Rampastring.Tools;
+
+namespace ClientCore.Extensions
+{
+    public static class RampastringToolsExtensions
+    {
+        public static string? GetStringValueOrNull(this IniSection section, string key) =>
+            section.KeyExists(key) ? section.GetStringValue(key, string.Empty) : null;
+
+        public static int? GetIntValueOrNull(this IniSection section, string key) =>
+            section.KeyExists(key) ? section.GetIntValue(key, 0) : null;
+
+        public static bool? GetBooleanValueOrNull(this IniSection section, string key) =>
+            section.KeyExists(key) ? section.GetBooleanValue(key, false) : null;
+
+        public static List<T>? GetListValueOrNull<T>(this IniSection section, string key, char separator, Func<string, T> converter) =>
+            section.KeyExists(key) ? section.GetListValue<T>(key, separator, converter) : null;
+
+
+    }
+}
```

Comment:
So we start to use extensions for Rampastring.Tool? Can you add `GetStringListValue` to trim and delete empty strings for future refactoring of random loading screens?

Reply:
I am not sure whether these codes should be included in the upstream `Rampastring.Tool` or as extensions here.

But one thing is for sure: we need such nullable methods to better implement the common settings for both the maps and the game modes

Reply:
> So we start to use extensions for Rampastring.Tool? Can you add `GetStringListValue` to trim and delete empty strings for future refactoring of random loading screens?

Can do in another PR. This PR already mixes a lot of code changes (although they are nearly all GameLobby/GameModeMap related)

Reply:
Ok, no problem. Wait this PR to merge.